app/controllers/concerns/application_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ApplicationController < ActionController::Base
+  helper_method :current_user
+  def current_user
+    @current_user ||= User.first
+  end
+end
```

Comment:
- [ ] Kindly remove the current_user method in your  ApplicationController, Devise provides a method to set your current user 👍🏿